Datasource: Make sure data proxy timeout applies to HTTP client (#25865)
grafana/grafana: docs/sources/administration/configuration.md

### timeout

> How long the data proxy should wait before timing out. Default is `30` (seconds).

Reviewer: ```suggestion
How long the data proxy should wait before timing out. Default is 30 seconds.
```

> > **Note:** This setting also applies to core backend HTTP data sources where queries are requested using an HTTP client with timeout set.

Author: ```suggestion
This setting also applies to core backend HTTP data sources where query requests use an HTTP client with timeout set.
```

### External user management

> If you manage users externally you can replace the user invite button for organizations with a link to an external site together with a description.

Reviewer: ```suggestion
If you manage users externally, then you can replace the user invite button for organizations with a link to an external site together with a customized description.
```

Author: Please handle this in a separate PR
